staged-dependencies.R

```diff
@@ -56,7 +60,7 @@ if (threads == "auto") {
 }
 
 # Install the remotes package
-if (!require("remotes", quietly = TRUE)) {
+if (!require("remotes", quietly = sd_quiet)) {
```

Comment:
This won't work. You need type conversion here.

Reply:
yes, I have it in line 41
`sd_quiet <- isTRUE(as.logical(Sys.getenv("SD_QUIET", "true")))`